runtime/kusama/src/lib.rs

```diff
@@ -408,8 +408,8 @@ parameter_types! {
 	// 1 hour session, 15 minutes unsigned phase, 8 offchain executions.
 	pub OffchainRepeat: BlockNumber = UnsignedPhase::get() / 8;
 
-	/// We take the top 12500 nominators as electing voters..
-	pub const MaxElectingVoters: u32 = 12_500;
+	/// We take the top 12500/22500 nominators as electing voters..
+	pub const MaxElectingVoters: u32 = prod_or_fast!(12_500, 25_000);
```

Comment:
This doesn't really make sense. If a zombinet is being launched based on Kusama, it should respect its limit. 25000 nominators, from which 12_500 being elected is reasonable. 

This won't help us test this for the scale of Polkadot though.

Reply:
Note that this is one-time test and we're testing here the limits including allocation limits. So having more nominators than what's *currently* on Kusama seems reasonable to me.

Reply:
Thanks for the clarification, I will revert this changes 👍